syft/messaging/message.py

```diff
@@ -54,21 +54,29 @@ def contents(self):
     def _simplify(self):
         return (self.contents,)
 
+    def __str__(self):
+        """Return a human readable version of this message"""
+        return f"({type(self).__name__} {self.contents})"
```

Comment:
self.contents will mostly be None because we'll use child classes, don't you think?
Do we want to keep the contents for Message?

Reply:
I tried to get rid of it, but bad things happened. Each message should probably have its own string representation, I suppose.

Reply:
Fixed this by making `Message` an Abstract Base Class, removing the `contents` property, and making each sub-class responsible for its own string representation.